VisualActivityViewController.swift

```diff
@@ -183,8 +183,10 @@ final class VisualActivityViewController: UIActivityViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         
-        UIView.animate(withDuration: fadeOutDuration) {
-            self.preview?.alpha = 0
+        UIView.animate(withDuration: fadeOutDuration, animations: {
+            self.preview?.alpha = 0;
+        }) {_ in
```

Comment:
```suggestion
        }) { _ in
```
Sorry for being picky, but can you add a space on the left side of the underscore?

Reply:
No problem. 1 minute.